script.sh

```diff
@@ -78,26 +79,71 @@ else
   echo "Press [Enter] to continue" && read answer
 fi
 
-echo "Extracting contents of downloaded IPMIView archive..."
-if [[ -d Contents/Resources/IPMIView ]]; then
-  rm -rf Contents/Resources/IPMIView
+echo "Using Java from $JAVA_HOME"
+arch=$( arch )
+if [ "$arch" = "x86_64" ] ; then
+  echo "Okay, using x86-64 platform."
```

Comment:
```suggestion
  echo "Using x86-64 platform."
```

recommend staying consistent with the below messaging format.